jenkins/test-history/gen_html.py

```diff
@@ -46,19 +46,26 @@
 ])
 
 
+BUCKET_TO_PREFIX = {}
 PREFIX_TO_URL = {}
 
-def prefix_for(prefix):
-    if 'azure' in prefix:
-        new = 'azure$'
-    elif 'rktnetes' in prefix:
-        new = 'rktnetes$'
-    elif 'gs://kubernetes-jenkins/' in prefix:
-        new = ''
-    else:
-        new = prefix[5:-1].replace('/', '_') + '$'
-    PREFIX_TO_URL[new] = prefix
-    return new
+
+def load_buckets_info(infile):
+    BUCKET_TO_PREFIX.clear()
+    if infile is None:
```

Comment:
Good point.  It would be true if None was passed as the second argument to main().  Although that's a possibility, that's not really a code path in how main() is currently used.

I debated whether "--buckets" should have a default value, be a required parameter, or be an optional parameter (in which case the default prefix would be used, line 66).  This is leftover from changing my mind.  I can remove this check.  Then, None or an invalid filename, will fail similarly - with an exception raised by open().